Fallout2/Fallout1in2/mods/fo1_german/text/german/dialog/SET.MSG

```diff
@@ -222,29 +222,22 @@
 {980}{Set206}{Daran w�rde ich nie denken.}
 {981}{Set310}{Meiner nicht w�rdig.}
 {982}{Set311}{Nicht mal im Traum!}
+{983}{Set204}{Hast du Stroh im Kopf?}
 
-{1000}{}{Meister}
-{1001}{}{Ghuls}
-{1002}{}{Mutanten}
-{1003}{}{Necropolis}
-{1004}{}{Krieg}
+{1000}{}{Meister|Mutantenanf�hrer|Anf�hrer}
+{1001}{}{Ghuls|Ghul}
+{1002}{}{Mutanten|Mutant|Muties}
+{1003}{}{Necropolis|Nekropolis|CITY OF DEAD|CITY OF THE DEAD}
+{1004}{}{Krieg|2077|Holocaust}
 {1005}{}{Einheit}
 {1006}{}{Hub}
-{1007}{}{Brotherhood of Steel}
-{1008}{}{Wasserstelle}
-{1009}{}{Children of the Cathedral}
-{1010}{}{Steuerchip}
-{1011}{}{Halle}
-{1012}{}{Children}
-{1013}{}{Garret}
-{1014}{}{Set}
-{1015}{}{Muties}
-{1016}{}{Brotherhood}
-{1017}{}{Wasserversorgung}
-{1018}{}{Cathedral}
-{1019}{}{Children of the Cathedral}
-{1020}{}{Stelle}
-{1021}{}{Wasser}
+{1007}{}{Brotherhood of Steel|Brotherhood|Steel|BOS}
+{1008}{}{Wasserstelle|Wasserversorgung|Wasser|Stelle}
+{1009}{}{Cathedral|Children|Children of the Cathedral|CoC}
+{1010}{}{Steuerchip|Chip|Wasserchip}
+{1011}{}{Halle|Halle der Toten|Totenhalle|Halle des Todes}
+{1012}{}{Garret}
+{1013}{}{Set|Seth}

```

Comment:
and again

Reply:
This is on purpose. I've added them because they are easy to misstype.

Reply:
You criminal dev :(

Reply:
Or lazy/misremembering people... I can't tell how many times I've called Set Seth in my mind (although I would write them correctly and wonder "wait, why did I call him Set(h)?"
We're all getting older :^